Mathlib/RingTheory/Flat/Stability.lean

```diff
@@ -18,7 +20,8 @@ We show that flatness is stable under composition and base change.
                       then `M` is a flat `R`-module
 * `Module.Flat.baseChange`: if `M` is a flat `R`-module and `S` is any `R`-algebra,
                             then `S ⊗[R] M` is `S`-flat.
-
+* `Module.Flat.isLocalizedModule_ofFlat`: if `M` is a flat `R`-module and `S` is a submonoid of `R`
```

Comment:
```suggestion
* `Module.Flat.of_isLocalizedModule`: if `M` is a flat `R`-module and `S` is a submonoid of `R`
```

Reply:
These results are stated as `R_p` flat. Can you make it clear in the docstring?
It's not hard to provide `R`-flatness as well (Use `isLocalizedModule_iff_isLocalization` and what you have here to get `R_p` flat over `R` and then use `Module.Flat.comp`) and it would be nice if you can do it in this PR but I'm also perfectly fine with it if it comes later.

Reply:
I don't think we have `R_p`is flat as an `R`-module, unless I'm missing something, so I'd rather postpone this.

Reply:
`isLocalizedModule_iff_isLocalization` and the newly added `Module.Flat.of_isLocalizedModule` implies that `Rp` is flat over `R`.